src/remote-config.ts

```diff
@@ -3,6 +3,7 @@ import remoteConfig from '@react-native-firebase/remote-config';
 export type RemoteConfig = {
   enable_ticketing: boolean;
   enable_intercom: boolean;
+  enable_i18n: boolean;
```

Comment:
Good idea! We should do this more often, that is to say feature toggling big features in case of regressions or extended testing. Love it! 🙌

Reply:
wee :smile: